.github/config.yml

```diff
@@ -0,0 +1,10 @@
+ # Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Thanks for your contribution and congrats on opening this pull request! ✨ 
+  
+  The dbt Labs docs are written in Markdown and sometimes HTML. When writing content, refer to the [style guide](https://github.com/dbt-labs/docs.getdbt.com#:~:text=refer%20to%20the-,style%20guide,-and%20content%20types) and [content types](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-types.md) to learn more about our writing standards and how we organize information in the product docs.
```

Comment:
```suggestion
We use Markdown and some HTML to write the dbt product documentation. When writing content, you can use our [style guide](https://github.com/dbt-labs/docs.getdbt.com#:~:text=refer%20to%20the-,style%20guide,-and%20content%20types) and [content types](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-types.md) to understand our writing standards and how we organize information in the dbt product docs.
```

Reply:
@mirnawong1 I'm not sure if the link is working exactly right, specifically the `-and%20content%20types`
 part of the `https://github.com/dbt-labs/docs.getdbt.com#:~:text=refer%20to%20the-,style%20guide,-and%20content%20types`
 
 We could link to the Style Guide directly, but open to this method.